lib/omniauth/strategies/facebook.rb

```diff
@@ -9,18 +9,15 @@ module Strategies
     class Facebook < OmniAuth::Strategies::OAuth2
       class NoAuthorizationCodeError < StandardError; end
 
-      DEFAULT_SCOPE = 'email'
+      DEFAULT_SCOPE = 'email'.freeze
+      API_VERSION = 'v2.6'.freeze
```

Comment:
@printercu Can you remove these `.freeze` calls -- inconsistent with other string constants in this gem.